docker/docker-compose.yml

```diff
@@ -139,7 +139,7 @@ services:
     ports:
       - 9216:9261
     command:
-      - "--mongodb.uri=mongodb://root:root@mongo:27017"
+      - "--mongodb.uri=mongodb://${MONGODB_USERNAME}:${MONGODB_PASSWORD}@mongo:27017"
```

Comment:
oh.my.god did we really overlook that?

Reply:
Yeeeeeep. It's been in the back of my mind for a while, but I kept forgetting about it.